frontend/src/views/CreditsPage/creditsPage.scss

```diff
@@ -0,0 +1,25 @@
+.creditsPage {
+  padding: 5%;
+}
+
+.githubButtons {
+  margin-top: 160px;
+  width: 30%;
+  gap: 20px;
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+  justify-content: center;
+  justify-items: center;
+  align-items: center;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  gap: 5%;
+}
+
+.creditsPageImg {
+  height: 600px;
+}
```

Comment:
We want to follow the convention so:

please rename the file creditsPage.scss to CreditsPage.scss
Additionally let's follow the CSS naming convention from App.scss (use hypen-separate-names instead of camelCase) eg. `github-buttons`
One last thing - those styles may cause problems when the rest of the code is merged - someone can have a `container` as well.  Try to create more specific selectors (eg. .credits-page .container).

Reply:
![image](https://user-images.githubusercontent.com/74785654/149620325-e4799618-3536-455a-a219-552e37c33aef.png)

Reply:
I have no idea why, but in the project, the name of the file is "CreditsPage.scss", whereas git cannot see it and says that it is "creditsPage.scss". Also, in CreditsPage.jsx, this file is imported as "CreditsPage.scss".

Reply:
check discord message :D